### PR TITLE
jsdom test

### PR DESCRIPTION
**Pull request by lbarjak**

### CODE REVIEW

File: site/js/unittests/jsdom-test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var jsdom = require("jsdom").jsdom;
```
Comment (magwas) [inline site/js/unittests/jsdom-test.js]: please use import

Comment (lbarjak) [inline site/js/unittests/jsdom-test.js]: import jsdom from "jsdom";
unfortunately does not work with node 7.7.1

**Comment by lbarjak** [comment]
+ assertion

**Comment by lbarjak** [comment]
integrating command line js testing framework
#950

File: site/js/unittests/jsdom-test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+var document = jsdom("<html><body></body></html>");
+var p = document.createElement("p");
+document.body.appendChild(p);
+var str = "******** jsdom DOM test ********"
+document.getElementsByTagName("p").innerHTML = str;
+var res = document.getElementsByTagName("p").innerHTML;
+
+QUnit.module( "test" ); 
+QUnit.test( "jsdom DOM test", function( assert ) {
+	assert.equal(str, res, "na" );
```
Comment (magwas) [inline site/js/unittests/jsdom-test.js]: leve out the message,or use something meaningful

**Pull request merged**